lib/api/html/functions.php

```diff
@@ -235,63 +235,64 @@ function beans_selfclose_markup_e( $id, $tag, $attributes = array() ) {
 }
 
 /**
- * Register close markup by ID.
+ * Build the closing HTML element's markup.  This function fires 2 separate hooks:
  *
- * This function is similar to {@see beans_open_markup()}, but does not accept HTML attributes. The $id
- * argument must be the identical to the opening markup.
+ *      1. "{id}_append_markup" - which fires first before the closing tag.
+ *      2. "{$id}_after_markup" - which fires after the closing tag.
  *
  * Note: You can pass additional arguments to the functions that are hooked to <tt>$id</tt>.
  *
  * @since 1.0.0
  *
  * @param string $id  Identical to the opening markup ID.
- * @param string $tag The HTML tag.
+ * @param string $tag The HTML tag. If set to False or empty, the markup HTML tag will be removed but
+ *                    the actions hook will be called. If set the Null, both markup HTML tag and actions
+ *                    hooks will be removed.
```

Comment:
Should False and Null be capitalized here?
`If set to False or empty, the HTML tag markup will be removed but the actions hook will be called.
If set to Null, both HTML tag markup and actions hook will be removed.`